components/org.wso2.carbon.identity.application.authenticator.biometric/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/notification/handler/impl/FirebasePushNotificationSenderImpl.java

```diff
@@ -91,9 +91,9 @@ public void sendPushNotification(String deviceId, String pushId,
 
             JSONObject biometricNotificationInfo = new JSONObject();
             biometricNotificationInfo.put(BiometricAuthenticatorConstants.BODY, message);
-            biometricNotificationInfo.put(BiometricAuthenticatorConstants.CONTENT_AVAILABLE, true);
-            biometricNotificationInfo.put(BiometricAuthenticatorConstants.PRIORITY,
-                    BiometricAuthenticatorConstants.HIGH);
+//            biometricNotificationInfo.put(BiometricAuthenticatorConstants.CONTENT_AVAILABLE, true);
+//            biometricNotificationInfo.put(BiometricAuthenticatorConstants.PRIORITY,
```

Comment:
Remove commented code. check for other places as well

Reply:
Changes for this class will be made in the next PR when the classes get updated as the push authenticator